src/TensorFlowNET.Keras/Engine/Model.Evaluate.cs

```diff
@@ -63,31 +66,76 @@ public partial class Model
             });
             callbacks.on_test_begin();
 
-            IEnumerable<(string, Tensor)> logs = null;
+            //Dictionary<string, float>? logs = null;
+            var logs = new Dictionary<string, float>();
```

Comment:
Because I wasn't very familiar with C# at the time of my last commit, so I change the definition of logs this time to align with the form of Model.fit,